pkg/operator/controller/externaldns/deployment_test.go

```diff
@@ -3377,3 +3440,45 @@ func testPlatformStatusGCP(projectID string) *configv1.PlatformStatus {
 		},
 	}
 }
+
+func TestBuildSecretHash(t *testing.T) {
```

Comment:
Can you please move this golang test function to the other golang test functions?
Just to keep it a bit organized because now we have: `TestX`, `TestY`, `testDeployment`, `testSecret`, `TestZ`.

Reply:
@alebedev87 Where exactly do you suggest that I move this test function?

Reply:
Moved it here https://github.com/openshift/external-dns-operator/pull/127/files#diff-fb0a8674418558ba280aa06686d117445fa00bf83fdca12efc520d8836460089R3174

Reply:
Just where I though, thanks a lot, @DhritiShikhar!